ports/atmel-samd/boards/pewpew10/mpconfigboard.mk

```diff
@@ -23,3 +23,5 @@ CIRCUITPY_USB_MIDI = 0
 SUPEROPT_GC = 0
 
 FROZEN_MPY_DIRS += $(TOP)/frozen/pew-pewpew-standalone-10.x
+
+CFLAGS_BOARD = --param inline-unit-growth=15 --param max-inline-insns-auto=15
```

Comment:
How about factoring these out for all small boards?

Reply:
I ended up with 3 different sets of flags for 5 boards.  My gut said this wasn't worth worrying about DRY.

+CFLAGS_BOARD = --param inline-unit-growth=15 --param max-inline-insns-auto=15
+CFLAGS_BOARD = --param inline-unit-growth=15 --param max-inline-insns-auto=12
+CFLAGS_BOARD = --param inline-unit-growth=15 --param max-inline-insns-auto=15
+CFLAGS_BOARD = --param inline-unit-growth=15 --param max-inline-insns-auto=20
+CFLAGS_BOARD = --param inline-unit-growth=15 --param max-inline-insns-auto=20

Reply:
Seems like the first param can be factored out. Adding new small builds will be easier if these CFLAGS automatically apply.